Sources/Card/Models/CardOrderApproval.swift

```diff
@@ -4,7 +4,7 @@ import PaymentsCore
 #endif
 
 /// The result of a card payment flow.
-public struct CardResult {
+public struct CardOrderApproval {
```

Comment:
I think this name also feels confusing for a first time merchant integrating. Returning an object that ends with the word `Approval` makes it seem like they are done with the order when in reality they need to take this result and either authorize or capture the order after they get this result back.

Reply:
Totally agree. There's a lot of good symmetry with `CardRequest` and `CardResult` as well.

Reply:
I went ahead and reverted to clean this up and maybe revisit down the line if it makes sense.